snap_stats.js

```diff
@@ -2,6 +2,7 @@ const fs = require('fs');
 const mustache = require('mustache');
 
 const templateName = 'template.html';
+const removeEmptyUsers = true;
```

Comment:
Please add a comment about what an 'empty user' is, or find a more explicit name.